open_src/influx/httpd/config/config.go

```diff
@@ -37,71 +37,73 @@ const (
+	BindAddress                   string         `toml:"bind-address"`
+	AuthEnabled                   bool           `toml:"auth-enabled"`
+	WeakPwdPath                   string         `toml:"weakpwd-path"`
+	LogEnabled                    bool           `toml:"log-enabled"`
+	SuppressWriteLog              bool           `toml:"suppress-write-log"`
+	WriteTracing                  bool           `toml:"write-tracing"`
+	FluxEnabled                   bool           `toml:"flux-enabled"`
+	FluxLogEnabled                bool           `toml:"flux-log-enabled"`
+	PprofEnabled                  bool           `toml:"pprof-enabled"`
+	DebugPprofEnabled             bool           `toml:"debug-pprof-enabled"`
+	HTTPSEnabled                  bool           `toml:"https-enabled"`
+	HTTPSCertificate              string         `toml:"https-certificate"`
+	HTTPSPrivateKey               string         `toml:"https-private-key"`
+	MaxRowLimit                   int            `toml:"max-row-limit"`
+	MaxConnectionLimit            int            `toml:"max-connection-limit"`
+	SharedSecret                  string         `toml:"shared-secret"`
+	Realm                         string         `toml:"realm"`
+	UnixSocketEnabled             bool           `toml:"unix-socket-enabled"`
+	UnixSocketGroup               *toml.Group    `toml:"unix-socket-group"`
+	UnixSocketPermissions         toml.FileMode  `toml:"unix-socket-permissions"`
+	BindSocket                    string         `toml:"bind-socket"`
+	MaxBodySize                   int            `toml:"max-body-size"`
+	AccessLogPath                 string         `toml:"access-log-path"`
+	AccessLogStatusFilters        []StatusFilter `toml:"access-log-status-filters"`
+	MaxConcurrentWriteLimit       int            `toml:"max-concurrent-write-limit"`
+	MaxEnqueuedWriteLimit         int            `toml:"max-enqueued-write-limit"`
+	EnqueuedWriteTimeout          toml.Duration  `toml:"enqueued-write-timeout"`
+	MaxConcurrentQueryLimit       int            `toml:"max-concurrent-query-limit"`
+	MaxEnqueuedQueryLimit         int            `toml:"max-enqueued-query-limit"`
+	QueryRequestRateLimit         int            `toml:"query-request-ratelimit"`
+	WriteRequestRateLimit         int            `toml:"write-request-ratelimit"`
+	EnqueuedQueryTimeout          toml.Duration  `toml:"enqueued-query-timeout"`
+	TLS                           *tls.Config    `toml:"-"`
+	WhiteList                     string         `toml:"white_list"`
+	SlowQueryTime                 toml.Duration  `toml:"slow-query-time"`
+	ParallelQueryInBatch          bool           `toml:"parallel-query-in-batch-enabled"`
+	QueryMemoryLimitEnabled       bool           `toml:"query-memory-limit-enabled"`
+	ChunkReaderParallel           int            `toml:"chunk-reader-parallel"`
+	ReadBlockSize                 toml.Size      `toml:"read-block-size"`
+	DisableQueryWithoutTimeFilter bool           `toml:"disable-query-without-time-filter"`
```

Comment:
Use global variables, see `querySeriesLimit ` or `LogRowsRuleSwitch `

https://github.com/openGemini/openGemini/blob/a18abae75d436e25df56d124176c8fb532ee37bb/lib/syscontrol/syscontrol.go#L93

https://github.com/openGemini/openGemini/blob/a18abae75d436e25df56d124176c8fb532ee37bb/lib/syscontrol/syscontrol.go#L115

Reply:
change to:
```
DisableNoTimeFilter bool           `toml:"disable-query-without-time-filter"`
```